src/components/Day.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import Todo from "./Todo";
+
+const DayContainer = styled.div`
+  font-family: "Acme", sans-serif;
+  font-family: "Caveat", cursive;
+  font-family: "Gochi Hand", cursive;
+  font-family: "Quintessential", cursive;
```

Comment:
font-family는 순서대로 적용되는 것이라, 한 번만 선언해줘도 되리라 봅니다.
그나저나 소나무 취향이시군요. 🤔

```suggestion
  font-family: Acme, Caveat, "Gochi Hand", Quintessential, sans-serif;
```